content/posts/genezio_ferretdb_tutorial.md

```diff
@@ -0,0 +1,145 @@
+---
+title: Use FerretDB with Genezio
+date: 2024-09-25
+tags:
+    - Tutorials
+author: Costin Sin
+linkedIn: https://www.linkedin.com/in/costin-sin/
+thumbnail: /images/TODO.webp
+preview: TODO
+# meta data start
+description: TODO
+meta_og_url: "https://genezio.com/blog/use-ferretdb-with-genezio/"
+meta_og_image: "https://genezio.com/images/TODO.webp"
+# meta data end
+customHeader: "White header"
+customFooter: "White footer"
+readTime: 12
+url: /blog/use-ferretdb-with-genezio/
+---
+
+FerretDB is an open-source alternative to MongoDB that allows you to use MongoDB drivers seamlessly with a PostgreSQL as the database backend. It acts like a proxy that understands the MongoDB wire protocol and translates the requests to SQL queries that are executed on the PostgreSQL database.
+
+This tutorial will guide you through setting up FerretDB with Genezio Functions and connecting to a PostgreSQL database provided by Genezio.
+
+## Prerequisites
+
+-   A Genezio account. Follow this link to [sign up](https://app.genez.io/auth/signup?utm_source=genezio&utm_medium=blog&utm_campaign=ferretdb).
+-   Clone or open in the browser the [FerretDB example repository](https://github.com/Genez-io/ferretdb-example)
```

Comment:
Access to the [FerretDB example repository](https://github.com/Genez-io/ferretdb-example).